cdap-ui/app/cdap/components/DataPrep/ColumnActionsDropdown/index.js

```diff
@@ -199,8 +208,8 @@ export default class ColumnActionsDropdown extends Component {
     let tableContainer = document.getElementById('dataprep-table-id');
 
     const tetherOption = {
-      attachment: 'top left',
-      targetAttachment: 'bottom right',
+      attachment: 'top right',
+      targetAttachment: 'bottom left',
```

Comment:
Tried both, seems like when `targetAttachment` is `bottom left` the dropdown hides less of the column